pwa/src/templates/templateParts/searchComponent/SearchComponentTemplate.tsx

```diff
@@ -40,7 +40,6 @@ export const SearchComponentTemplate: React.FC = () => {
           {t("Search")}
         </Button>
         <Button
-          className={styles.submit}
```

Comment:
This seems like an unintentional change.

Reply:
Because it was among console errors